### PR TITLE
Update TomoPhantom and CCPi regularisation tags

### LINKED ISSUE #812: Update CCPi Regularisation and TomoPhantom tags
The tags for CCPi Regularisation and TomoPhantom have not kept up to date with the current latest stable releases.

### PR DESCRIPTION
**Pull request by paskino**
closes #812

### CODE REVIEW

**Comment by KrisThielemans** [comment]
I've merged `master`. Possibly the tags are again already out-of-date of course.

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
please update `CHANGES.md`

**Comment by KrisThielemans** [comment]
CIL_regularisation tests failing with stuff like
```
  File "/home/runner/work/SIRF-SuperBuild/SIRF-SuperBuild/build/sources/CCPi-Regularisation-Toolkit/test/test_run_test.py", line 389, in test_TGV_CPU_vs_GPU
    (tgv_gpu, info_vec_gpu) = TGV(pars['input'],
  File "/home/runner/install/python/ccpi/filters/regularisers.py", line 117, in TGV
    return TGV_GPU(inputData,
NameError: name 'TGV_GPU' is not defined
```

File: SuperBuild/External_CCPi-Regularisation-Toolkit.cmake [hunk 1]
```diff
@@ -98,8 +98,8 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
 
     set(${proj}_ROOT        ${${proj}_SOURCE_DIR})
     set(${proj}_INCLUDE_DIR ${${proj}_SOURCE_DIR})
-    add_test(NAME CIL_REGULARISATION_TEST_1
-             COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -s test -p test_*.py
+    add_test(NAME CIL_REGULARISATION_TEST_CPU
+             COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -s test -p test_CPU_*.py
```
Comment (KrisThielemans) [inline SuperBuild/External_CCPi-Regularisation-Toolkit.cmake:102]: can't we discover if there's a GPU (I think we have a CMake variable for it)? If not, then this is fine.

Comment (paskino) [reply SuperBuild/External_CCPi-Regularisation-Toolkit.cmake:102]: We have a variable `${${proj}_USE_CUDA}` but it only detects if CUDA SDK is present not the availability of a GPU.

Comment (paskino) [reply SuperBuild/External_CCPi-Regularisation-Toolkit.cmake:102]: At any rate this should be handled upstream, https://github.com/vais-ral/CCPi-Regularisation-Toolkit/issues/177

**Review by KrisThielemans — APPROVED** [review]
let's squash-merge this one. No idea why there are so many commits here!

**Pull request merged**